core/pyproject.toml

```diff
@@ -50,6 +50,8 @@ pymssql = { version = "^2.2.7", optional = true }
 PyMySQL = { version = "^1.0.2", optional = true }
 redis = { version = "^4.3.5", optional = true }
 SQLAlchemy = { version = "^2.0.4", optional = true }
+kafka-python = { version = "^2.0.2", optional = true }
+confluent-kafka = { version= "^2.3.0", optional = true }
```

Comment:
Do we need them both?

Reply:
Yes - for TestContainer you need kafka-python, as it turned out while I ran tests in docker container emulating github